packages/client/lib/client/index.ts

```diff
@@ -867,10 +874,11 @@ export default class RedisClient<
   }
 
   MULTI() {
-    type Multi = new (...args: ConstructorParameters<typeof RedisClientMultiCommand>) => RedisClientMultiCommandType<[], M, F, S, RESP, TYPE_MAPPING>;;
+    type Multi = new (...args: ConstructorParameters<typeof RedisClientMultiCommand>) => RedisClientMultiCommandType<[], M, F, S, RESP, TYPE_MAPPING>;
     return new ((this as any).Multi as Multi)(
       this._executeMulti.bind(this),
-      this._executePipeline.bind(this)
+      this._executePipeline.bind(this),
+      this._self.#options
```

Comment:
?

Reply:
ok, with what we discussed yesterday, putting the handler on object via attachCommand, this is no longer neccessary, so reverted.